app/src/main/java/org/oppia/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -110,11 +111,13 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
               activity,
               internalProfileId
             )
+
+          // added a flag to intent so that the activity created only once and used if already existed,
```

Comment:
Please follow the conventions of other line comments for consistency (e.g. capitalize the the sentence & describe what the code does rather than the actions you took here).

Reply:
done